### DISCUSSION TITLE
Kanban Board

### DISCUSSION THREAD

**Discussion by MrPentha** [EveronLife/EveronLife#57] [Polls]

**Comment by Tomboru** [comment]
Maybe just the "Projects" thing from GitHub?
But I would agree that we need something like this to get a better overview.

**Comment by Arkensor** [comment]
A better overview of what exactly? The base stages of development should be outlined by where you see them. Discussion -> Issue -> PR -> main. A board would be for issues, but issues will only represent things in development, so what sub-states would you like to point out?

The issues right now are all in the same phase because of the pre-planning and concepts already made. So they are all under development right now. Maybe that causes some confusion?

**Reply by MrPentha** [reply]
You can show the different phases of development in Github, but as far as I know you don't have an overview of the entire project. The stages in the Kanban board can be named exactly like in Github

**Comment by Jaccodouma** [comment]
This may be related to a discussion in discord, I think the goal is to provide a better overview on who is (actively) working on what issues. It might be nice to have people leave a reply on issues with something along the lines of "This issue is being worked on by x on [this](https://github.com/EveronLife/EveronLife/discussions/57) fork." so people have a sense on what's being worked on and who to contact if they want to help or something.

**Reply by Arkensor** [reply]
Yeah, this will probably all be much more clear once things are all rolling as they are supposed to. If an issue has people assigned to it, they are responsible for everything, as outlined in the contribution guide. We can have a board that shows what issues are under development and which are in testing, that might make sense.
